web/src/utils/state/atoms.ts

```diff
@@ -46,10 +46,10 @@ export const isWindLayerEnabledAtom = atom(
     get(selectedDatetimeIndexAtom).index === 24
 );
 
-export const solarLayerLoadingAtom = atom(false);
-export const windLayerLoadingAtom = atom(false);
+export const solarLayerLoadingAtom = atom<boolean>(false);
+export const windLayerLoadingAtom = atom<boolean>(false);
```

Comment:
Just curious why the type was added here?

Reply:
I was adding a type below where it couldn't be inferred and thought it was also kinda nice to have them explicit where possible.